### PR TITLE
Replace all CSS/l10n selectors with data-test-id for unit tests

### LINKED ISSUE #8734: Replace all CSS and l10n selectors with data-test-id for unit tests
This would ease the work of Aphrodite refactoring since we lose the option to tag components by CSS selector.

This work is split between unit tests and webdriver tests.

### PR DESCRIPTION
**Pull request by cezaraugusto**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Close #8734
Auditors: @bsclifton, @nejczdovc
Test Plan: `npm run unittest` should pass

### CODE REVIEW

File: app/renderer/components/preferences/payment/ledgerTable.js [hunk 1]
```diff
@@ -182,6 +182,7 @@ class LedgerTable extends ImmutableComponent {
             disabled
             checkedOn
             indicatorClassName={css(styles.pinnedToggle)}
+            testId='disabled'
```
Comment (NejcZdovc) [inline app/renderer/components/preferences/payment/ledgerTable.js]: Can you be more specific here, maybe `pinnedDisabled`?

File: app/renderer/components/common/messageBox.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ class MessageBox extends React.Component {
   }
 
   render () {
-    return <Dialog className={css(styles.dialog)}>
+    return <Dialog testId='dialog' className={css(styles.dialog)}>
```
Comment (NejcZdovc) [inline app/renderer/components/common/messageBox.js]: Can you be more specific here? Maybe `messageBoxDialog`?

File: app/renderer/components/common/messageBox.js [hunk 3]
```diff
@@ -138,6 +138,7 @@ class MessageBox extends React.Component {
           {
             this.props.showSuppress
               ? <SwitchControl
+                testId='switchControl'
```
Comment (NejcZdovc) [inline app/renderer/components/common/messageBox.js]: Can you be more specific here? Maybe `showSuppressSwitch`?

**Comment by bsclifton** [comment]
@cezaraugusto can you rebase when you get a chance? Then it's ready for merge 😄

**Pull request merged**